package.json

```diff
@@ -23,17 +23,32 @@
   "peerDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.58.0",
     "@typescript-eslint/parser": "^5.58.0",
-    "eslint": "^8.38.0",
+    "eslint": ">=8.38.0",
```

Comment:
Source of change: https://eslint.org/docs/latest/extend/shareable-configs#publishing-a-shareable-config